README.md

```diff
@@ -1,72 +1,40 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# capstone project
 
+> In this capstone project I managed to build the website based on an online website for a conference. using the provided design guidelines to create the website, I personalized the content.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
```

Comment:
- [ ] Please add .vscode folder to your .gitognore file as it makes the project space unclean.